src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+$width-boxs: 100px;
+$width-gap: 1px;
+$calendar-days: 31;
+$days-of-the-week: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
+
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-boxs * 7 + 6 * $width-gap;
+  gap: $width-gap;
+  &__day {
+    box-sizing: border-box;
+    width: $width-boxs;
+    height: $width-boxs;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: all 0.5s;
```

Comment:
it's not a good practice to use `all`. specify what exactly you'd like to change